helmfile/charts/prometheus-alerts/templates/rules/backup-status.yaml

```diff
@@ -0,0 +1,128 @@
+        runbook_url: {{ .Values.defaultRules.runbookUrl }}alert-name-influxdbbackup
+        summary: The daily backup job influxdb-backup have failed over 48 hours.
+      expr: |-
+        (
+          (min((time()-kube_job_status_completion_time{job_name=~"influxdb-backup-[0-9]*", cluster=~".*"})/3600)) > 48
+        )
+      for: 1h
+      labels:
+        severity: critical
+    - alert: VeleroBackup
+      annotations:
+        description: The job daily backup job velero-backup have failed over 24 hours.
+        runbook_url: {{ .Values.defaultRules.runbookUrl }}alert-name-velerobackup
+        summary: The daily backup job velero-backup have failed over 24 hours.
+      expr: |-
+        (
+          (time() - velero_backup_last_successful_timestamp{schedule="velero-daily-backup", cluster=~".*"}) / 60 / 60 > 24
+        )
+      for: 1h
+      labels:
+        severity: warning
+    - alert: VeleroBackup
+      annotations:
+        description: The job daily backup job velero-backup have failed over 48 hours.
+        runbook_url: {{ .Values.defaultRules.runbookUrl }}alert-name-velerobackup
+        summary: The daily backup job velero-backup have failed over 48 hours.
+      expr: |-
+        (
+          (time() - velero_backup_last_successful_timestamp{schedule="velero-daily-backup", cluster=~".*"}) / 60 / 60 > 48
+        )
+      for: 1h
+      labels:
+        severity: critical
+    - alert: ElasticsearchBackup
+      annotations:
+        description: The job daily backup job elasticsearch-backup have failed over 12 hours.
+        runbook_url: {{ .Values.defaultRules.runbookUrl }}alert-name-elasticsearchbackup
+        summary: The daily backup job elasticsearch-backup have failed over 12 hours.
+      expr: |-
+        (
+          min((time()-kube_job_status_completion_time{job_name=~"elasticsearch-backup-.*", cluster=~".*"})/3600) > 12
+        )
+      for: 1h
+      labels:
+        severity: warning
+    - alert: ElasticsearchBackup
+      annotations:
+        description: The job daily backup job elasticsearch-backup have failed over 24 hours.
+        runbook_url: {{ .Values.defaultRules.runbookUrl }}alert-name-elasticsearchbackup
+        summary: The daily backup job elasticsearch-backup have failed over 24 hours.
+      expr: |-
+        (
+          min((time()-kube_job_status_completion_time{job_name=~"elasticsearch-backup-.*", cluster=~".*"})/3600) > 24
+        )
+      for: 1h
+      labels:
+        severity: critical
+    - alert: ElasticsearchSnap
+      annotations:
+        description: The job daily backup job {{`{{`}} $labels.repository {{`}}`}} have failed over 80 hours.
+        runbook_url: {{ .Values.defaultRules.runbookUrl }}alert-name-elasticsearchsnap
+        summary: The daily backup job elasticsearch snapshot have failed over 80 hours.
+      expr: |-
+        (
+          (time()-elasticsearch_snapshot_stats_snapshot_start_time_timestamp{state="SUCCESS", cluster=~".*"})/3600 > 80
```

Comment:
will this be triggered if `state=PARTIAL`?

Reply:
Apparently not

Reply:
if state="SUCCESS" doesn't exists the alert will not be triggered
I would like to also cover this scenario

Reply:
Added by adding a "or" statement.